tests/scripts/thread-cert/run_cert_suite.py

```diff
@@ -95,9 +97,6 @@ def cleanup_backbone_env():
 def setup_backbone_env():
     bash('sudo modprobe ip6table_filter')
 
-    if THREAD_VERSION != '1.2':
```

Comment:
why remove this check?

Reply:
because we are now using OTBR Docker for 1.1 certification tests.